content/docs-v1/multiformat-output.md

```diff
@@ -141,9 +131,19 @@ $print-entry-caption-color: $white;
 $print-entry-image-display: all; // first | all
 ```
 
-There are even more PDF-related variables that can be modified in the `content/_assets/styles/print.scss` file. Including rules regarding the running feet and page numbering that are included at the bottom of each PDF page.
+By default, the PDF is output at full-resolution and with crop marks for professional printing. Figure images are included in the PDF at the same size they were added to the `content/_assets/images/` directory, except when the images have been designated as zooming with `zoom: true` in the `figures.yaml` file. For zooming images, Quire processes and uses a special image that's 2025px wide for the PDF and EPUB output. These are large enough for a full-page image in a printed book. This value can be changed in `_plugins/figures/iiif/config.js`, for instance to make the images smaller for a web-ready PDF. PDFs can also be downsampled to a smaller file size after they have been generated, using a program like Adobe Acrobat.
```

Comment:
From "Figure images are included..." to "...Adobe Acrobat" doesn't feel like it belongs in this section which is essentially about variables. It feels weird when it switches back to variables when you get to "crop marks and bleed..." I could be misunderstanding something but I feel like this paragraph belongs elsewhere, perhaps a section dedicated to figures in PDFs?

Reply:
Yes, I see your point. I'd like to keep the figure resolution and crop marks stuff together because they're both about how the PDF comes out and what the intended end use is, print or web. So perhaps a new heading and/or it should move to a different part of the page?